practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,36 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: 5rem 5rem;
```

Comment:
I think you should use a unit in your app by using a consistent unit. If you would like to use rem, please define a conversion formula from 'px' to 'rem' in your code instead of using fixed values here